src/charts/legend.coffee

```diff
@@ -219,3 +228,10 @@ Ember.Charts.Legend = Ember.Mixin.create
       .text((d) -> d.label)
       .attr(@get 'legendLabelAttrs')
       .call(labelTrimmer.get 'trim')
+
+    # we use averageLegendLabelWidth in order to estimate how much to move the
+    # labels to make them seem roughly centered
+    totalLabelWidth = 0
+    legend.selectAll('text').each ->
+      totalLabelWidth += this.getComputedTextLength()
+    @set 'averageLegendLabelWidth', totalLabelWidth / @get 'legendItems.length'
```

Comment:
Oh OK, it's set here. It should probably be A) moved from where it is defined above (so as not to make it unclear if the user can set it) and B) turned into an Ember computed property if possible (can observe `legendItems.[]`)